src/api/entities/Checkpoint.ts

```diff
@@ -210,14 +220,18 @@ export class Checkpoint extends Entity<UniqueIdentifiers, HumanReadable> {
       u64ToBigNumber(checkpointId).gte(id)
     );
 
+    // If there has been a balance change since the Checkpoint was created, then we can query the Checkpoint storage
+    // If there hasn't been a balance update, the storage will not have an entry for the Identity. Instead the current balance should be queried instead.
```

Comment:
```suggestion
    /*
     * If there has been a balance change since the Checkpoint was created, then we can query the Checkpoint storage.
     * Otherwise, the storage will not have an entry for the Identity. The current balance should be queried instead.
     */
```